test/lib/jdk/test/lib/Platform.java

```diff
@@ -348,6 +349,20 @@ private static boolean isArch(String archnameRE) {
                       .matches();
     }
 
+    public static boolean isOracleLinux7() {
+        if (System.getProperty("os.name").toLowerCase().contains("linux") &&
+                System.getProperty("os.version").toLowerCase().contains("el")) {
```

Comment:
Hi,  I checked and the el ("Enterprise Linux" ?)  is shown in os.version on RHEL and Oracle Linux.  But how reliable is it? Could this show up also on some other distros in os.version (maybe in the cpu-related string of uname output?) ?

Reply:
I'm not aware of any conventions in the output of uname that would prevent other operating systems from including "el" followed by a number.

Reply:
okay , then let's keep it the way it is now and adjust later if needed.